mathesar_ui/src/sections/import-data/importUtils.ts

```diff
@@ -391,13 +391,18 @@ export function cancelImport(fileImportStore: FileImport): void {
   }
 }
 
-export async function importFromURL(fileImportStore: FileImport, url: string): Promise<void> {
+interface DataFilesRequestData {
+  url?: string,
+  paste?: string,
+}
+
+async function importData(fileImportStore: FileImport, data: DataFilesRequestData): Promise<void> {
   setInFileStore(fileImportStore, {
     uploadStatus: States.Loading,
```

Comment:
@seancolsen It works fine, but does it matter that ` uploadStatus` never equals `States.Done`?

Reply:
Good eye, @mr-gabe49! That's how the code was before this PR, but it does seem a little fishy doesn't it. It doesn't seem like we're relying on that `uploadStatus` anywhere that would break without setting it to `States.Done`, but for the sake of cleanliness I've added some code in 5e19618eb3b2ffd651f74a725a9ab77040f0a5c7 that sets the status to done.